### PR TITLE
[DEM-202] Modal directive and terms n conditions

### PR DESCRIPTION
**Pull request by vietnogi**
ping @appirio-tech/demandside

### CODE REVIEW

File: src/client/app/layout/styles/modal.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
 @import "../../core/styles/combined";
 
+#modal-overlay {
```
Comment (tladendo) [inline src/client/app/layout/styles/modal.scss]: should this really be an id? what if we want to have multiple modals on a page?

**Comment by tladendo** [comment]
lgtm besides comment

File: src/client/app/layout/styles/modal.scss (hunk 1 above)
Comment (vietnogi) [inline src/client/app/layout/styles/modal.scss]: I gave that some thought and decided to use an ID to force future developers to only use one modal at a time.  Using nested modals gets ugly fast and the user experience is not that friendly.

Comment (tladendo) [inline src/client/app/layout/styles/modal.scss]: hmm...won't this still allow for two (or more) modals on a page at once? i mean, all but one may be hidden, but that doesn't mean that they're not in the DOM.

Comment (vietnogi) [inline src/client/app/layout/styles/modal.scss]: this line should prevent that: https://github.com/appirio-tech/ap-work-client/pull/149/files#diff-8d2dfc258637d30cde01cc1e779329bcR8

Comment (tladendo) [inline src/client/app/layout/styles/modal.scss]: cool

**Pull request merged**